sqlmesh/core/snapshot/evaluator.py

```diff
@@ -1676,6 +1684,116 @@ def get_custom_materialization_type(name: str) -> t.Type[CustomMaterialization]:
     return strategy_type
 
 
+class EngineManagedStrategy(MaterializableStrategy):
+    def create(
+        self,
+        table_name: str,
+        model: Model,
+        is_table_deployable: bool,
+        render_kwargs: t.Dict[str, t.Any],
+        **kwargs: t.Any,
+    ) -> None:
+        is_snapshot_deployable: bool = render_kwargs["is_snapshot_deployable"]
```

Comment:
This should be `kwargs`, right?

Reply:
Ah, yes, since the merge that broke out `render_kwargs` and `kwargs` into two separate things. It looks like there is a missing test case, i'll add one

Reply:
Actually, this was in the `render_kwargs`, but i'll move it